elements/lisk-validator/src/validation.ts

```diff
@@ -242,3 +242,18 @@ export const validateFee = (data: string): boolean =>
 	isNumberString(data) &&
 	isGreaterThanZero(new BigNum(data)) &&
 	!isGreaterThanMaxTransactionAmount(new BigNum(data));
+
+export const isCsv = (data: string): boolean => {
+	const maxItemCount = 1000;
+	if (typeof data !== 'string') {
+		return false;
+	}
+
+	const csvAsArray = data.split(',');
+
+	if (csvAsArray.length > 0 && csvAsArray.length <= maxItemCount) {
+		return true;
+	}
+
+	return false;
```

Comment:
`isCsv` will return `false` for any CSV with more than `maxItemCount`.
Maybe use a better name for this function which express that restriction?

Reply:
I would prefer not to change this behaviour now as the implementation comes from framework and is used in the schemas there so I need to research first why it was like this in the first place. Maybe work it in a separate issue?